lib/BenchShow.hs

```diff
@@ -97,5 +99,15 @@ module BenchShow
     ) where
 
 import BenchShow.Common
-import BenchShow.Graph
 import BenchShow.Report
+
+#ifdef CHARTING
+import BenchShow.Graph
+#else
+graph :: FilePath -> FilePath -> Config -> IO ()
+graph _ _ _ =
+    putStrLn
+        $ unlines [ "Library was compiled without charting support."
+                  , "See the flags available in the cabal file for more info."
+                  ]
+#endif
```

Comment:
We can keep this module unchanged. If we want to use it without graphs we can import the lower level modules.

Reply:
What do you mean by unchanged? This needs to be changed if graphs do not exist right? Are you suggesting we move this CPP stuff into `Grpah` module?

Reply:
This can always include the Graph module. When we have `no-charts` flag on we can exclude this module altogether and just use the lower level module directly.